rclc/src/rclc/init.c

```diff
@@ -63,7 +64,8 @@ rclc_support_init_with_options(
     allocator, "allocator is a null pointer", return RCL_RET_INVALID_ARGUMENT);
   rcl_ret_t rc = RCL_RET_OK;
 
-  memcpy(&support->init_options, init_options, sizeof(rcl_init_options_t));
+  support->init_options = rcl_get_zero_initialized_init_options();
+  rcl_init_options_copy(init_options, &support->init_options);
 
   support->context = rcl_get_zero_initialized_context();
   rc = rcl_init(argc, argv, &support->init_options, &support->context);
```

Comment:
```suggestion
  support->context = rcl_get_zero_initialized_context();
  rc = rcl_init(argc, argv, init_options, &support->context);
```
`support->init_options` is not used anywhere on the rclc

Reply:
Related:
- https://github.com/ros2/rclc/pull/203